### DISCUSSION TITLE
Is there any way to highlight context while using find and replace ?

### DISCUSSION THREAD

**Discussion by MarvelousChen** [VSCodeVim/Vim#7368] [Q&A]
When using "find and replace" (eg. :%s/***/***/g), how to enable highlighting so we can justify conveniently if we typed correct match patterns?

**Comment by J-Fields** [comment]
Currently, no. Some infrastructure that would make this possible was implemented in 028e9c6533078729066d32ee3fee8d6f87986470, but I haven't gotten around to finishing the feature yet. You can track progress at #4934.

**Comment by MarvelousChen** [comment]
Thank you very much!
